docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -1,16 +1,16 @@
 // tag::ess[]
 
-{ecloud} runs a hosted version of {integrations-server}. No extra setup is
-required unless you want to scale your deployment.
+{ecloud} runs a hosted version of {integrations-server} that includes
+{fleet-server}. No extra setup is required unless you want to scale your
+deployment.
 
 To confirm that an {integrations-server} is available in your deployment:
 
-. Log in to {kib} and go to *Management > {fleet}*.
-. Click the *Agents* tab.
-. In the *Agent policy* column, look for *Elastic Cloud agent policy* --
-This policy is managed by {ecloud}, and contains a {fleet-server}
-integration and an Elastic APM integration. You cannot modify the policy.
-Confirm that the agent status is *Healthy*.
+. In {kib}, open the main menu, and go to *Management > {fleet}*.
+. On the *Agents* tab, look for the *Elastic Cloud agent policy*. This policy is
+managed by {ecloud}, and contains a {fleet-server} integration and an Elastic
+APM integration. You cannot modify the policy. Confirm that the agent status is
```

Comment:
The comma shouldn't be there if you follow the rule "don't put a comma between a subject and a verb." Sometimes I'll add an errant comma when there are multiple conjunctions, and I think it helps the readability of a sentence. This could be an argument for breaking the sentence down into proper independent clauses. :-) :-) 

I'll accept your suggestion.